### PR TITLE
🧹 refactor: Improve topic names for clarity and consistency

### LINKED ISSUE #344: [Bug]: Kafka warning on ch_events_json
### What happened?

Kafka throws a warning when creating the topic `ch_events_json`

`Kafka is ready now, start creating topics.
WARNING: Due to limitations in metric names, topics with a period ('.') or underscore ('_') could collide. To avoid issues it is best to use either, but not both.
Seed kafka topic finished.`

Other topics are named using - instead of _ so this also seems like a naming consistency problem.

### To Reproduce

`docker compose -f docker-compose-pro.yaml  up` or
`docker compose -f docker-compose-pro-dev.yaml  up`
Review logs and observe warning

### Expected behavior

no warning message should be created

### Additional context

This medium article has some background information [Kafka Topic Naming Conventions: Best Practices](https://medium.com/@kiranprabhu/kafka-topic-naming-conventions-best-practices-6b6b332769a3)

To avoid possible collisions and maintain consistency in topic names, I suggest renaming `ch_events_json` to `ch-events-json` 

minimal changes would be required in the following files.

featbit\modules\back-end\src\Domain\Messages\Topics.cs line: 11
featbit\modules\data-analytics\app\clickhouse\kafka\kafka_topics.py line: 3
featbit\modules\evaluation-server\src\Domain\Core\Topics.cs Line: 11
featbit\docker-compose-pro-dev.yml Line: 164
featbit\docker-compose-pro.yml Line: 147

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by dsun0720**
Fixed #344

### CODE REVIEW

File: modules/data-analytics/app/clickhouse/kafka/kafka_topics.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 from app.setting import KAFKA_PREFIX, SUFFIX
 
-KAFKA_EVENTS_TOPIC = f"{KAFKA_PREFIX}ch_events_json{SUFFIX}"
+KAFKA_EVENTS_TOPIC = f"{KAFKA_PREFIX}ch-events-json{SUFFIX}"
```
Comment (deleteLater) [inline modules/data-analytics/app/clickhouse/kafka/kafka_topics.py]: Why `KAFKA_PREFIX` and `SUFFIX`

**Comment by deleteLater** [comment]
Suggested new topic names
- evaluation-server-endusers => featbit-endusers
- api-feature-flag-change => featbit-feature-flag-change
- api-segment-change => featbit-segment-change
- ch_events_json => featbit-insights

**Review by deleteLater — APPROVED** [review]
LGTM

**Comment by cosmic-flood** [comment]
Chanting:Review PR

**Pull request merged**